### PR TITLE
update g_lineplot with cohort param.

### LINKED ISSUE #1077: update g_lineplot to count cohorts
related to [tlg issue #131](https://github.com/insightsengineering/tlg-catalog/issues/131)

### PR DESCRIPTION
**Pull request by ayogasekaram**
closes #1077

### CODE REVIEW

**Review by Melkiades — COMMENTED** [review]
Thanks Abi! Could you add a small regression test?

File: tests/testthat/_snaps/summarize_coxreg.md [hunk 1]
```diff
@@ -147,15 +147,15 @@
...
-          40                          0.62       (0.42, 0.93)                                
+        40                            0.62       (0.42, 0.93)                                
         A Covariate Label   340                                                 0.9883       
-          1                           0.63       (0.35, 1.14)                                
-          2                           0.58       (0.27, 1.26)                                
-          3                           0.55       (0.22, 1.35)                                
-          4                           0.69       (0.23, 2.07)                                
+        1                             0.63       (0.35, 1.14)                                
+        2                             0.58       (0.27, 1.26)                                
+        3                             0.55       (0.22, 1.35)
```
Comment (Melkiades) [inline tests/testthat/_snaps/summarize_coxreg.md:154]: Why these changed? Check if you have the latest rtables-formatters

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Abi :)

**Comment by legrasv** [comment]
@ayogasekaram Fine for me, I tried the function, it worked well !!

**Pull request merged**